Add question 079 about scheduled workflows in GitHub Actions (#225)
FidelusAleksander/ghcertified: content/questions/actions/question-080.md

---
archetype: "questions"
title: "Question 080"

> question: "which is the correct syntax for setting the directory for all jobs in a workflow?"

Reviewer: ```suggestion
question: "What is the correct syntax for setting the directory for all `run` commands in a workflow?"
```

Author: thanks!
